solution/Makefile

```diff
@@ -6,14 +6,44 @@ PWD := $(shell pwd)
 BUILD_ROOT ?= $(PWD)/_build
 ISO_ROOT ?= $(BUILD_ROOT)/root
 
+# Metadata
+PRODUCT_NAME := Zenko
+PRODUCT_LOWERNAME := zenko
+BUILD_TIMESTAMP := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
+BUILD_HOST := $(shell hostname)
+
+ifneq '$(RELEASE)' '1'
+DEVELOPMENT_RELEASE = 1
+SUFFIX ?= -dev
+else
+DEVELOPMENT_RELEASE = 0
+endif
+
+# Use git tag as our version
+VERSION_SHORT := $(shell git describe --abbrev=0)
+VERSION_FULL := $(VERSION_SHORT)$(SUFFIX)
+GIT_REVISION := $(shell git describe --long --always --tags --dirty)
+
 # Solution default configuration
 config: $(ISO_ROOT)/config.yaml
 .PHONY: config
-
 $(ISO_ROOT)/config.yaml: $(PWD)/config.yaml
 	@mkdir -p $(@D)
 	cp -f $< $@
 
+# Solution metadata product.txt
+product: $(ISO_ROOT)/product.txt
+.PHONY: product
+$(ISO_ROOT)/product.txt:
+	@mkdir -p $(@D)
+	echo NAME=${PRODUCT_NAME} > $@
```

Comment:
Did you set those variables in the shell environment? Otherwise I'm surprised this is working, I think you need the `$()` syntax to access Makefile variables.

Reply:
Make supports either $() or ${}, according to docs and my testing. They are just set from the top of the file.

Reply:
Ok, good to know